opba-banking-rest-impl/src/test/java/de/adorsys/opba/tppbankingapi/mapper/FacadeResponseBodyToRestBodyMapperTest.java

```diff
@@ -30,4 +54,30 @@ public void testMapping() {
         assertEquals(IBAN, restEntity.getAccounts().get(0).getIban());
         assertEquals(DELETED, restEntity.getAccounts().get(0).getStatus().name());
     }
+
+    @Test
+    @SneakyThrows
+    void accountsMapperTest() {
+        URL inputUrl = Resources.getResource("mapperTestFixtures/facade_to_rest_response_accounts_mapper_input.json");
+        AccountListBody inputObject = JSON_MAPPER.readValue(inputUrl, AccountListBody.class);
+        AccountList outputObject = accountsRestMapper.map(inputObject);
+
+        URL outputUrl = Resources.getResource("mapperTestFixtures/facade_to_rest_response_accounts_mapper_output.json");
+        AccountList expectedObject = JSON_MAPPER.readValue(outputUrl, AccountList.class);
+
+        assertThat(expectedObject).isEqualToComparingFieldByField(outputObject);
+    }
+
+    @Test
+    @SneakyThrows
+    void transactionsMapperTest() {
+        URL inputUrl = Resources.getResource("mapperTestFixtures/facade_to_rest_response_transactions_mapper_input.json");
```

Comment:
Wrap resource reading and mapping to some helper function

Reply:
ok